sdk/src/account.rs

```diff
@@ -14,6 +15,19 @@ pub struct Account {
     pub executable: bool,
 }
 
+impl fmt::Debug for Account {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "Account {{ tokens: {} userdata.len: {} owner: {} executable: {} }}",
+            self.tokens,
+            self.userdata.len(),
```

Comment:
How about a base58 string instead for userdata? Check out the instance for Pubkey.

Reply:
how is it helpful though?

Reply:
You can diff it.  Also, the extremely motivated can copy that string and pass it to a utility that parses it to recreate the original account.

Reply:
I feel like if you want something useful, you add another debug at some point where you have deserialized the userdata and then print something that makes sense depending on the account type. It's way less work to do that than it is to write a whole new utility to parse strings that you copy and paste and better debug experience.

Reply:
The default Debug instances include *all* the information, but it's too verbose.  The next closest thing is *all* the information in a more compact way, like base58. If that's still too verbose (which it's generally not), then printing just the length makes sense to me.

Reply:
A hex string is another, probably better option.

Reply:
Here's a compromise, up to the first 64 bytes hex-formatted if it's not 0 and always include the length.